content/id/docs/reference/networking/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Referensi Jaringan
+content_type: reference
+weight: 85
+---
+
+<!-- overview -->
+Di bagian ini tersedia referensi detail mengenai jaringan komputer di Kubernetes.
```

Comment:
```suggestion
Bagian dari dokumentasi Kubernetes ini menyediakan referensi detail tentang jaringan Kubernetes.
```

Reply:
Thanks for the review. Let me change @habibrosyad

Reply:
Revised done @habibrosyad